cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -483,8 +483,33 @@ def create(self, catalog_item_class, *args, **kwargs):
         Returns:
             An instance of catalog_item_class
         """
+        if kwargs:
```

Comment:
this check is unnecessary, kwargs will be defined, and you're using `get` to reference expected keys.